RestaurantReservation.API/Models/Customer/CustomerUpdateDto.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace RestaurantReservation.API.Models.Customer;
+
+/// <summary>
+/// Data transfer object for updating a customer.
+/// </summary>
+public class CustomerUpdateDto
```

Comment:
I do not think you need to define two separate DTOs for create and update if all the properties are the same. u can use a single  general DTO instead